index.html

```diff
@@ -124,7 +124,8 @@ <h1>LSST DESC registration form </h1>
       <div class="form-check">
         <input class="form-check-input" type="checkbox" id="contact" name="contact">
         <label class="form-check-label" for="contact">
-        Check if you do NOT wish to be considered for the role of meeting contact person.
+        Check if you do NOT wish to be considered for the role of 
+          <a href="http://lsstdesc.org/sites/default/files/LSST_DESC_Professional_Conduct.pdf#page=11" target="_top">Meeting Contact Persons</a>.
```

Comment:
Thanks for the change, actually, Rachel pointed out that it's not just a question of willingness but also of restrictions on who can be a contact person:  “… or are not eligible to serve in this role”

Reply:
So the worry is that people may not know whether or not they are eligible.  The MeetComm will remove those who are not eligible via an automated script. (cc @rmandelb)